common/persistence/cassandra/nexus_service_store.go

```diff
@@ -34,6 +36,40 @@ import (
 	"go.temporal.io/server/common/persistence/nosql/nosqlplugin/cassandra/gocql"
 )
 
+const (
+	UnknownTableVersion = 0
```

Comment:
I don't think we need this constant, just use 0 throughout the code IMHO.

Reply:
Also no reason to export this AFAICT.

Reply:
My thinking is it will be needed at the manager level, but I will un-export it for now